src/scripts/main.js

```diff
@@ -1,3 +1,85 @@
 'use strict';
 
 // write code here
+const appendRowBut = document.querySelector('.append-row');
+const removeRowBut = document.querySelector('.remove-row');
+const appendColBut = document.querySelector('.append-column');
+const removeColBut = document.querySelector('.remove-column');
+const table = document.querySelector('.field tbody');
+
+appendRowBut.addEventListener('click', (e) => {
+  e.preventDefault();
+
+  if (table.rows.length === 10) {
+    appendRowBut.disabled = true;
+  }
+
+  const row = document.querySelector('tr');
+  const newRow = row.cloneNode(true);
```

Comment:
The line `const row = document.querySelector('tr');` selects the first `<tr>` element in the document, which might not be the intended behavior if the table has multiple rows. Consider selecting a specific row from the table, such as the last row, to clone.